internal/pkg/levee/propagation/propagation.go

```diff
@@ -51,9 +57,6 @@ func Dfs(n ssa.Node, conf source.Classifier, taggedFields fieldtags.ResultType)
 	return record
 }
 
-// dfs performs Depth-First-Search on the def-use graph of the input Source.
```

Comment:
Why remove this comment? I think the second part of the comment was very useful for a newb like me. Did the behavior mentioned in the comment change?

Reply:
The last sentence is actually incorrect. When the traversal encounters a Sanitizer, it merely records that the sanitizer has been reached. Sanitizers are not otherwise treated differently from other calls.

Removing the comment is the wrong thing to do here, though. I have rewritten the comment instead, thank you for your input! WDYT about this new comment?

Reply:
LGTM! Thanks for adding great documentation to this method.